cmd/monaco/main.go

```diff
@@ -83,7 +85,15 @@ func RunImpl(args []string, fileReader util.FileReader) (statusCode int) {
 	util.Log.Info("Dynatrace Monitoring as Code v" + version.MonitoringAsCode)
 
 	apis := createApis()
-
+	if input.downloadFlag {
+		statusCode = download.GetConfigs(input.environments, input.path, input.downloadSpecificAPI)
+	} else {
+		statusCode = syncConfigs(input.dryRun, input.environments, input.projectNameToDeploy, apis, input.path, fileReader, deploymentErrors)
+	}
+	return statusCode
+}
+func syncConfigs(dryRun bool, environments map[string]environment.Environment, projectNameToDeploy string,
```

Comment:
That step should still be 'deployConfigs' (or similar) - we don't technically synchronize, we push the local config over anything on the cluster